scripts/dandiset_000004_converted.yaml

```diff
@@ -0,0 +1,242 @@
+- affiliations:
+  - Department of Neurology, Cedars-Sinai Medical Center, Los Angeles, CA, USA
+  email: Jeffrey.Chung@cshs.org
+  name: Jeffrey M. Chung
+  roleName:
+  - Author
+  - ProjectMember
+  - Validation
+- affiliations:
+  - Department of Neurology, Cedars-Sinai Medical Center, Los Angeles, CA, USA
+  email: Chrystal.Reed@csmc.edu
+  name: Chrystal M. Reed
+  roleName:
+  - Author
+  - ProjectMember
+  - Validation
+- affiliations:
+  - Biological Systems & Engineering Division, Lawrence Berkeley National Laboratory,
+    Berkeley, CA, USA
+  - Department of Neurosurgery, Stanford University, Stanford, CA, USA
+  email: ben.dichter@gmail.com
+  name: Ben Dichter
+  roleName:
+  - Author
+  - Software
+  - ProjectMember
+  - Validation
+- affiliations:
+  - Institute for Interdisciplinary Brain and Behavioral Sciences, Crean College of
+    Health and Behavioral Sciences, Schmid College of Science and Technology, Chapman
+    University, Orange, CA, USA
+  - Division of Biology and Biological Engineering, California Institute of Technology,
+    Pasadena, CA, USA
+  email: maoz.uri@gmail.com
+  name: Uri Maoz
+  roleName:
+  - Author
+  - Conceptualization
+  - ProjectMember
+  - Validation
+- affiliations:
+  - Division of Neurosurgery, Department of Surgery, University of Toronto, Toronto,
+    Canada
+  - Krembil Brain Institute, Toronto Western Hospital, Toronto, Canada
+  email: suneil.kalia@uhn.ca
+  name: Suneil K. Kalia
+  roleName:
+  - Author
+  - ProjectMember
+  - Validation
+- affiliations:
+  - Krembil Brain Institute, Toronto Western Hospital, Toronto, Canada
+  - Division of Neurosurgery, Department of Surgery, University of Toronto, Toronto,
+    Canada
+  email: Taufik.Valiante@uhn.ca
+  name: Taufik A. Valiante
+  roleName:
+  - Author
+  - ProjectMember
+  - Validation
+- affiliations:
+  - Department of Neurosurgery, Cedars-Sinai Medical Center, Los Angeles, CA, USA
+  email: Adam.Mamelak@cshs.org
+  name: Adam N. Mamelak
+  roleName:
+  - Author
+  - ProjectMember
+  - Validation
+- affiliations:
+  - Department of Neurosurgery, Cedars-Sinai Medical Center, Los Angeles, CA, USA
+  - Department of Neurology, Cedars-Sinai Medical Center, Los Angeles, CA, USA
+  - Division of Biology and Biological Engineering, California Institute of Technology,
+    Pasadena, CA, USA
+  - Computational and Neural Systems Program, California Institute of Technology,
+    Pasadena, CA, USA
+  - Center for Neural Science and Medicine, Department of Biomedical Science, Cedars-Sinai
+    Medical Center, Los Angeles, CA, USA
+  email: Ueli.Rutishauser@cshs.org
+  identifier:
+    identifier: 0000-0002-9207-7069
+    identifierType: ORCID
+  name: Ueli Rutishauser
+  roleName:
+  - Author
+  - Conceptualization
+  - FundingAcquisition
+  - ProjectMember
+  - Resources
+  - Software
+  - Supervision
+  - Validation
+- awardNumber: U01NS103792
+  name: National Institute of Neurological Disorders and Stroke
+  roleName: &id001
```

Comment:
this rolename does not look right

Reply:
will look into what happened there.

Reply:
unfortunately, this is proper yaml. so instead of repeating they are assigning a reference id `&id001` to the first time roleName sponsor is being listed and referencing it in the other rolenames `*id001`.